cmd/cluster-olm-operator/main.go

```diff
@@ -161,6 +167,13 @@ func runOperator(ctx context.Context, cc *controllercmd.ControllerContext) error
 		}(c)
 	}
 
+	for _, c := range clusterCatalogControllerList {
+		go func(c factory.Controller) {
+			defer runtime.HandleCrash()
+			c.Run(ctx, 1)
+		}(c)
+	}
```

Comment:
We put this after the sleep so that the `ClusterCatalog` CRD is established before we attempt to interact with the API?